docker-entrypoint-initdb.d/initdb.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+POSTGRES="gosu postgres"
```

Comment:
I'm not sure I understand the purpose of this file. Everything that's done in here should be done already by [the startup script in `docker-library/postgres`](https://github.com/docker-library/postgres/blob/master/9.5/docker-entrypoint.sh), except possibly creation of the test database, which we could probably document separately...

...or am I misunderstanding something?